packages/devtools_app/lib/src/memory/memory_controller.dart

```diff
@@ -275,13 +275,15 @@ class MemoryController extends DisposableController
   }
 
   /// Automatic pruning of memory statistics (plotted) full data is still retained.
+  static const displayDefault = 'Default';
```

Comment:
what does default mean? Also, consider making these an enum like class with a pretty string name and a Duration object.

Reply:
Default is the best view each tick is 10 pixels, the width of an event symbol e.g., snapshot, monitor, etc.

Added this to the definition.

Added a TODO to add the enum with the duration.